src/repository/listRepository.ts

```diff
@@ -325,7 +325,8 @@ export const listRepository = {
         .where('listId', listId)
         .leftJoin('mediaItem', 'listItem.mediaItemId', 'mediaItem.id')
         .modify((qb) => filterQuery(qb, args))
-        .modify((qb) => sortQuery(qb, args))
+        //Dont need to sort when just doing a count(*)!
```

Comment:
Actually, sorting by 'next-airing' removes items that do not have an 'next-airing', so filterQuery is required. I know, it's not a great design :)